docs/020-deployment/howtos/111-aks-upgrade.md

```diff
@@ -0,0 +1,216 @@
+
+1. Open up `pwsh` 
+1. Run the following commands:
+
+    ```powershell
+    $aksParams = @(
+        '--name', ${aksClusterName} # This is the name of the AKS cluster
+        '--resource-group', ${resourceGroup} # This is the resource group name the AKS cluster resides in
+        '--subscription', ${subscription} # This is the subscription the AKS cluster resides in
+    )
+
+    $command = 'kubectl scale statefulset -n cluedin --replicas=0 --all --timeout 5m'
+    az aks command invoke @aksParams --command $command
+
+    $command = 'kubectl scale deploy -n cluedin --replicas=0 --all --timeout 5m'
+    az aks command invoke @aksParams --command $command
+    ```
+
+    The above may take a couple minutes to complete. But once done, it should be safe to upgrade the AKS cluster without any potential data loss.
+
+1. Because CluedIn uses Kubernetes API, normally during upgrade time you may face an error when attempting to do the AKS upgrade. As a result, it will prevent you from upgrading without bypassing the API check first. 
+To disable this API check, run the following command:
+
+    ```powershell
+    $params = @(
+        '--name', ${aksClusterName}
+        '--resource-group', ${resourceGroup}
+        '--subscription', ${subscription}
+        '--enable-force-upgrade'
+        '--upgrade-override-until', ((Get-Date).AddDays(1).ToString('yyyy-MM-ddT00:00:00Z'))
+    )
+    az aks update @params
+    ```
+
+    This will take a few minutes to complete and should return back successfully
+
+With the above steps completed, it's time to upgrade. Depending if you pass or fail the pre-requisites, please follow the appropriate path below.
+
+[Passed](#i-have-passed-the-azure-checks-above)   
+[Failed](#i-have-not-passed-the-azure-checks-above)
+
+### I have passed the Azure Checks above
+
+Because everything checks out from the Azure side, you should be able to upgrade the Kubernetes cluster without any issues. Please note that Kubernetes follows an upgrade path and it's not always possible to upgrade from a lower version to a much higher version. You may need to perform this operation a number of times.
+
+### Upgrade Cluster
+1. Determine what version of Kubernetes you will need to get to, and then perform the below steps until you are successfully running that version.
+1. Open up `pwsh`
+1. Run the following commands:
+
+    ```powershell
+    $params = @(
+        '--name', ${aksClusterName}
+        '--resource-group', ${resourceGroup}
+        '--subscription', ${subscription}
+        '--kubernetes-version', '1.nn.n' # This will vary depending on what CluedIn version you are running.
+        '--yes'
+    )
+    az aks upgrade @params
+    ```
+1. It will take between 5-15 minutes to complete each upgrade. Once you have completed the upgrade path to the necessary version, skip down to the bottom to scale up your environment again. [Post-upgrade](#post-upgrade)
```

Comment:
```suggestion
1. It will take between 5-15 minutes to complete each upgrade. Once you have completed the upgrade path to the necessary version, scale up your environment again following the steps [here](#post-upgrade).
```